src/full-promises.test.ts

```diff
@@ -0,0 +1,82 @@
+import { ClientSecurity } from "./util";
+import * as v1 from "./v1";
+import { Consistency } from "./v1";
+import * as v1alpha from "./v1alpha1";
+import { generateTestToken } from './__utils__/helpers' 
+
+describe("a check following a write of schema and relationships", () => {
+  it("should succeed", async (done) => {
+    // Write the schema.
+    const token = generateTestToken('full-promises')
+    const alphaClient = v1alpha.NewClient(token, "localhost:50051", ClientSecurity.INSECURE_LOCALHOST_ALLOWED);
```

Comment:
Switch this to using v1 for writing schema, now that v1alpha1 is deprecated

Reply:
@josephschorr for this test, it creates an `alphaClient` and a `v1Client` similarly to the [`full.test.ts`](https://github.com/authzed/authzed-node/pull/47/files/742028b528add5c02040ce913e0ded43ab5877f9#diff-161cc4236742d31916058df4634edbf83f43ad791a2e7d251bd384fa6ab1936a) (appears as a smoke test to make sure they both work) - I can remove the `alphaClient` altogether, if that would be preferred.

Reply:
👍 sounds great, I'll update that now.